all.json

```
@@ -27,6 +27,12 @@
 	],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"tokens-connect.surekequipmentservice.com",
```

```suggestion
		"tokens-connect.surekequipmentservice.com",
		"surekequipmentservice.com",
```

Google also has this as deceptive.